funnel/views/email.py

```diff
@@ -139,6 +147,37 @@ def send_email_for_organization_admin_membership_revoked(
     )
 
 
+@signals.user_registered_for_project.connect
+def send_email_for_project_registration(project, user, locale=None):
+    background_project_email.queue(
+        project_id=project.id, user_id=user.id, locale=get_locale()
+    )
+
+
+@rq.job('funnel')
+def background_project_email(project_id, user_id, locale=None):
+    EmailAttachment = namedtuple('EmailAttachment', ['file', 'filename', 'mimetype'])
+    project = Project.query.get(project_id)
+    user = User.query.get(user_id)
+    with app.app_context():
+        send_email(
+            subject=_("Registration confirmation for {project}").format(
+                project=project.title
+            ),
+            to=[user],
+            attachments=[
+                EmailAttachment(
+                    file=schedule_ical(project),
```

Comment:
Rename this field to `content`. Use `file` only if it's a file-like object. Over here you have a string.